app/scripts/models/auth_brokers/oauth.js

```diff
@@ -61,6 +61,12 @@ function (_, Constants, Url, OAuthErrors, AuthErrors, p, Validate,
       afterSignIn: new HaltBehavior()
     }),
 
+    defaultCapabilities: _.extend({}, proto.defaultCapabilities, {
+      // Disable inter-tab sign-in for OAuth due to the potential for unintended
+      // consequences from redirecting to a relier URL more than once.
+      interTabSignIn: false
```

Comment:
Just to understand, `interTabSignIn` only affects the sign in message, all environments support the signout message by default?